core/src/main/scripts/importer/validateData.py

```diff
@@ -1360,19 +1360,25 @@ def skipValidation(self, data):
         entrez_id = '0'
         if 'Entrez_Gene_Id' in self.cols:
             entrez_id = data[self.cols.index('Entrez_Gene_Id')]
-        if hugo_symbol == 'Unknown' and entrez_id == '0' and variant_classification != 'IGR':
-            # the MAF specification documents the use of Unknown and 0 here
-            # for intergenic mutations, and since the Variant_Classification
-            # column is often invalid, cBioPortal interprets this combination
-            # (or just the symbol if the Entrez column is absent) as such,
-            # but with a warning:
-            self.logger.warning(
-                "Gene specification for this mutation implies "
-                "intergenic even though Variant_Classification is "
-                "not 'IGR'; this variant will be filtered out",
-                extra={'line_number': self.line_number,
-                       'cause': "Gene symbol 'Unknown', Entrez gene id 0"})
+        if hugo_symbol == 'Unknown' and entrez_id == '0':
             is_silent = True
+            if variant_classification == 'IGR':
+                self.logger.info("cBioPortal does not support variants with Variant Classification 'IGR'. "
```

Comment:
Please remove "does not support", since strictly speaking this is not completely accurate. IGR would still be loaded in a specific configuration (i.e. with custom filtering configured to not filter IGR and if entrez/hugo are set to valid value).

Reply:
Thanks, I fixed it.